File: src/oneadvisor/web/src/config/scope.ts
export type Scope = {
    id: number;
    name: string;
};

export const getScopes = (): Scope[] => {
    return [
        {
            id: 1,
            name: "Organisation",
        },
        {
            id: 2,
            name: "Branch",
        },
        {
            id: 3,
            name: "User",
        },
    ];
};

File: src/OneAdvisor.Model/Directory/Model/User/Scope.cs
namespace OneAdvisor.Model.Directory.Model.User
{
    public enum Scope
    {
        Organisation = 1,
        Branch = 2,
        User = 3
    }

    public class ScopeParser
    {
        public static Scope Parse(string scope)
        {
            switch (scope)
            {
                case "organisation":
                    return Scope.Organisation;
                case "branch":
                    return Scope.Branch;
                default:
                    return Scope.User;
            }
        }

        public static string Format(Scope scope)
        {
            switch (scope)
            {
                case Scope.Organisation:
                    return "organisation";
                case Scope.Branch:
                    return "branch";
                default:
                    return "user";
            }
        }
    }


}